src/Workspaces/Remote/ServiceHub/Shared/ServiceHubServiceBase.cs

```diff
@@ -29,6 +29,7 @@ internal abstract class ServiceHubServiceBase : IDisposable
         private Checksum _solutionChecksumOpt;
         private RoslynServices _lazyRoslynServices;
 
+        [Obsolete("For backward compatibility. this will be removed once all callers moved to new ctor")]
```

Comment:
LUT/Razor both require this change to go in to change their code. I just existing behavior so that they don't get broken until they move to new one.